server/ServerFSM.cpp

```diff
@@ -455,6 +459,21 @@ MPLobby::MPLobby(my_context c) :
                     player_setup_data.m_starting_species_name = sm.SequentialPlayableSpeciesName(player_id);
 
                 m_lobby_data->m_players.push_back(std::make_pair(player_id, player_setup_data));
+            } else if (player_id != Networking::INVALID_PLAYER_ID && player_connection->GetClientType() == Networking::CLIENT_TYPE_AI_PLAYER) {
+                PlayerSetupData player_setup_data;
+                player_setup_data.m_player_id =     -1;
```

Comment:
why the hard-coded / magic `-1` and not `Networking::INVALID_PLAYER_ID`?

Reply:
Fixed.